subsys/nfc/lib/platform_internal.h

```diff
@@ -0,0 +1,43 @@
+/*
```

Comment:
Maybe create an include folder and put there this header file. Next in CMakeLists.txt you can add this include folder as a private library include directory